.github/workflows/update-dist.yml

```diff
@@ -0,0 +1,27 @@
+on: [pull_request]
```

Comment:
what about forks?

Reply:
Updated workflow to only run on release PRs (autorelease: pending) so that the dist files are only updated on release.